dvclive/live.py

```diff
@@ -23,7 +25,11 @@ class Live:
     DEFAULT_DIR = "dvclive"
 
     def __init__(
-        self, path: Optional[str] = None, resume: bool = False, **kwargs
+        self,
+        path: Optional[str] = None,
+        resume: bool = False,
+        report: bool = True,
```

Comment:
See also https://github.com/iterative/dvclive/issues/91#issuecomment-1029388277. Users shouldn't have to touch their code when going from local to cml, so I think we probably want this set through an environment variable or config file.

Reply:
> See also https://github.com/iterative/dvclive/issues/91#issuecomment-1029388277. Users shouldn't have to touch their code when going from local to cml, so I think we probably want this set through an environment variable or config file.

`html` currently uses env var because it relies on DVC integration. Once we extract the render and DVCLive is in charge of generating the HTML, would you still expect it to be handled by env var?

> I think we should look at local html reports and cml markdown reports as two different report formats, which could be extended to others. See https://github.com/iterative/dvclive/issues/90#issuecomment-1035120185.

I like the approach of considering`html` and `cml` as just different formats of `report`. However, I have some doubts, like would it make sense to support multiple report formats in a single `Live` instance?

I'm not sure about the env var, I understand the reasoning but we could make code the top priority while still supporting the env var scenario for CI/CD with CML:

```
report: Optional[typing.Literal["cml", "html"] ] = None
. . . 
if report is None:
    report = os.getenv("DVCLIVE_REPORT", "html")
```

This would preserve current behavior locally and allow to enable CML report via env var on CI/CD